### PR TITLE
FBX-134 export skinned mesh if exporting blendshape animation

### PR DESCRIPTION
**Pull request by vkovec**
make sure to export the mesh and blendshapes in animation only export mode.

### CODE REVIEW

File: com.unity.formats.fbx/Editor/FbxExporter.cs [hunk 1]
```diff
@@ -2877,6 +2877,12 @@ private FbxNode CreateFbxNode(GameObject unityGo, FbxScene fbxScene)
                         ExportLight (go, fbxScene, node);
                     } else if (compType == typeof(Camera)) {
                         ExportCamera (go, fbxScene, node);
+                    } else if (compType == typeof(SkinnedMeshRenderer))
```
Comment (benoithudson) [inline com.unity.formats.fbx/Editor/FbxExporter.cs:2880]: Either follow the convention in this function of
```
} else if (...) {
```
or follow the Unity convention of 
```
}
else if (...)
{
```
but not halfway!

File: com.unity.formats.fbx/Editor/FbxExporter.cs [hunk 2]
```diff
@@ -2877,6 +2877,12 @@ private FbxNode CreateFbxNode(GameObject unityGo, FbxScene fbxScene)
...
+                    {
+                        // export only what is necessary for exporting blendshape animation
+                        var unitySkin = go.GetComponent<SkinnedMeshRenderer>();
+                        var meshInfo = new MeshInfo(unitySkin.sharedMesh, unitySkin.sharedMaterials);
```
Comment (benoithudson) [inline com.unity.formats.fbx/Editor/FbxExporter.cs:2884]: check for a missing `sharedMesh` here

Comment (benoithudson) [reply com.unity.formats.fbx/Editor/FbxExporter.cs:2884]: unless ExportMesh already does

Comment (vkovec) [reply com.unity.formats.fbx/Editor/FbxExporter.cs:2884]: there's an IsValid check in ExportMesh

**Pull request merged**